modules/platforms/cpp/ignite/client/detail/sql/result_set_impl.h

```diff
@@ -154,15 +154,15 @@ class result_set_impl : public std::enable_shared_from_this<result_set_impl> {
      *
      * @return @c true if there are more pages with results and @c false otherwise.
      */
-    [[nodiscard]] IGNITE_API bool has_more_pages() { return m_resource_id.has_value() && m_has_more_pages; }
+    [[nodiscard]] bool has_more_pages() { return m_resource_id.has_value() && m_has_more_pages; }
```

Comment:
Is `IGNITE_API` removed because this is `_imp.h`?

Reply:
Yeah, those methods should not be in API